plugins/embark-assistant/embark-assistant.cpp

```diff
@@ -314,7 +314,19 @@ command_result embark_assistant(color_ostream &out, std::vector <std::string> &
             embark_assist::main::state->survey_results[i][k].survey_completed = false;
             embark_assist::main::state->survey_results[i][k].neighboring_clay = false;
             embark_assist::main::state->survey_results[i][k].neighboring_sand = false;
-            embark_assist::main::state->survey_results[i][k].neighboring_aquifer = embark_assist::defs::Clear_Aquifer_Bits;
+            for (uint8_t l = 0; l <= ENUM_LAST_ITEM(biome_type); l++) {
```

Comment:
There's also a FOR_ENUM_ITEMS macro in DataDefs.h, in case you were unaware - you're perfectly welcome not to use it, though (for stylistic reasons or otherwise - it does yield values of the enum type instead of the base integer type, although I think either would work here).

Reply:
Thanks. I didn't know that.